src/main/java/com/github/fge/jsonpatch/CopyOperation.java

```diff
@@ -43,6 +43,8 @@
 public final class CopyOperation
     extends DualPathOperation
 {
+    public static final String OPERATION_NAME = "copy";
+
     @JsonCreator
     public CopyOperation(@JsonProperty("from") final JsonPointer from,
         @JsonProperty("path") final JsonPointer path)
```

Comment:
If we stick with the name in the constructor, use the const you defined above in the super("copy", ...) below.